src/main/java/com/arcbees/bourseje/client/admin/dashboard/AdminDashboardView.java

```diff
@@ -52,7 +52,7 @@
     @UiField
     Element currentState;
     @UiField
-    HTMLPanel candidates;
+    SpanElement candidates;
```

Comment:
In fact, I put a span element because with a div, the "Add a candidate" after the candidates was necessarily on a new line, as with a span, it would respect nicely the layout and put right at the right place. But I'm open to suggestions @jasonlemay ?